.changeset/spicy-wombats-wash.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
```

Comment:
[suggest] `minor` should be fine since fixing false negatives will produce more lint problems.

```suggestion
"stylelint": minor
```